### PR TITLE
Add auth providers list and page to the EE login workflow

### PR DESCRIPTION
**Pull request by HichamELBSI**
### What does it do?

Describe the technical changes you did.

### Why is it needed?

Describe the issue you are solving.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

**Review by soupette — CHANGES_REQUESTED** [review]

File: packages/strapi-admin/ee/admin/containers/AuthPage/components/Login/index.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```
[the remark this answers is not in the document]
Comment (HichamELBSI) [reply packages/strapi-admin/ee/admin/containers/AuthPage/components/Login/index.js:1]: This rule isn't needed anymore as ProviderButton is now an `a` HTML element so the accessibility is natively activated (Navigated between providers with `tab`, press `enter` to select a provider).

**Comment by pelumiojebode** [comment]
I provide

File: packages/strapi-admin/ee/admin/containers/AuthPage/components/Login/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (HichamELBSI) [reply packages/strapi-admin/ee/admin/containers/AuthPage/components/Login/index.js:1]: This rule isn't needed anymore as ProviderButton is now an `a` HTML element so the accessibility is natively activated (Navigated between providers with tab, press enter to select a provider).

**Review by soupette — APPROVED** [review]
LGTM

**Review by soupette — APPROVED** [review]

**Pull request merged**